lib/matplotlib/ticker.py

```diff
@@ -2410,26 +2410,26 @@ def tick_values(self, vmin, vmax):
             ticklocs = []
             if decade_min <= -1:
                 expo = np.arange(decade_min, min(0, decade_max + 1))
-                ticklocs.extend(list(10**expo))
+                ticklocs.extend(10**expo)
             if decade_min <= 0 <= decade_max:
                 ticklocs.append(0.5)
-            if decade_max >= 1:
+            if 1 <= decade_max:
                 expo = -np.arange(max(1, decade_min), decade_max + 1)
-                ticklocs.extend(list(1 - 10**expo))
+                ticklocs.extend(1 - 10**expo)
 
         # minor ticks
         else:
             ticklocs = []
             if decade_min <= -2:
                 expo = np.arange(decade_min, min(-1, decade_max))
                 newticks = np.outer(np.arange(2, 10), 10**expo).ravel()
-                ticklocs.extend(list(newticks))
+                ticklocs.extend(newticks)
             if decade_min <= 0 <= decade_max:
                 ticklocs.extend([0.2, 0.3, 0.4, 0.6, 0.7, 0.8])
-            if decade_max >= 2:
+            if 2 <= decade_max:
```

Comment:
This is backwards to the way most people think.  I guess you want to write it this way so the numbers are in order, and I wouldn't hold up the PR on this, but...

Reply:
Sorry, that was from a while ago, undoing both of these...